Ollivanders/src/net/pottercraft/ollivanders2/spell/ALIQUAM_FLOO.java

```diff
@@ -87,7 +87,7 @@ protected void doCheckEffect ()
          Location statLocation = new Location(location.getWorld(), target.getX() + 0.5, target.getY() + 0.125, target.getZ() + 0.5);
 
          // find the sign above the fire
-         if (target.getRelative(BlockFace.UP).getType() == Material.WALL_SIGN)
+         if (common.materialIsASign(target.getRelative(BlockFace.UP).getType()))
```

Comment:
I think it's ok but I want to confirm that this is changing from checking for a wall sign to checking for any sign

Reply:
made it only allow wallsigns